BrainPortal/app/models/cluster_task.rb

```diff
@@ -1956,7 +1956,8 @@ def submit_cluster_job
         jobid              = scir_session.run(job)
         self.cluster_jobid = jobid
         self.status_transition(self.status, "Queued")
-        self.addlog("Queued as job ID '#{jobid}'.")
+        self.addlog("Queued as job ID '#{jobid}'" +
+            "with walltime #{job.walltime}s and extra args are '#{job.tc_extra_qsub_args.presence || 'not provided'}'." )
```

Comment:
1. add a space before the word 'with'
2. change 'not provided' to '(none provided)'